hack/install.sh

```diff
@@ -425,7 +425,7 @@ do_install() {
 			(
 			set -x
 			for key_server in $key_servers ; do
-				$sh_c "apt-key adv --keyserver hkp://${key_server}:80 --recv-keys ${gpg_fingerprint}" && break
+				$sh_c "apt-key adv --keyserver hkp://${key_server}:80 --keyserver-options http-proxy=$http_proxy --recv-keys ${gpg_fingerprint}" && break
```

Comment:
When I tested it, `apt-key` ignored this option when neither of these variables was set. But yeah, if only `$HTTP_PROXY` (uppercase) is set, this change is not sufficient